include/sound/sof.h

```diff
@@ -30,6 +30,16 @@ struct snd_sof_pdata {
 	/* parent device */
 	struct device *dev;
 
+	/*
+	 * notification callback used if the hardware initialization
+	 * can take time or is handled in a workqueue. This callback
+	 * can be used by the parent device to e.g. enable runtime_pm
+	 * or limit functionality until all low-level inits are
+	 * complete. The device argument refers to the parent, not the
```

Comment:
@plbossart "parent device" in this comment could be misconstrued as the parent of the PCI/ACPI device. Could we maybe say top-level SOF device instead? Same comment on line 30 above as well. Also if "sof-audio" is removed, we should not refer to it here which could lead someone to go looking for it.